replication-api-impl/src/main/java/com/connexta/replication/api/impl/data/ReplicationItemManagerImpl.java

```diff
@@ -59,11 +59,6 @@ public ReplicationItemManagerImpl(ItemRepository itemRepository, SolrTemplate so
     this.solrTemplate = solrTemplate;
   }
 
-  @Override
```

Comment:
❓ Same question about removing this one. Especially since the query service will need to create this object and should not create the impl directly